data/categories.txt

```diff
@@ -14789,6 +14828,46 @@ tr:İskele|Marina|Rıhtım
 uk:5Пристань|3Марина
 zh-Hans:游艇码头
 
+piste:type-downhill|piste:type-nordic
```

Comment:
Let's make a UI category for them like @food, there will be many other types.

Reply:
Maybe next time for the UI, not in this PR.